pkgs/development/python-modules/rubicon-objc/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, darwin
+, fetchPypi
+, importlib-metadata
+, pythonOlder
+, setuptools-scm
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "rubicon-objc";
+  version = "0.4.6";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-+qbjc96rk7pwiBNUuIQTe4BAPOePyiOA8YnLM4s1CSY=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = lib.optionals (pythonOlder "3.8") [
+    importlib-metadata
+  ];
```

Comment:
We don't have anything older than 3.8 anymore.

Reply:
Thanks for the suggestion.
I removed unnecessary `propagatedBuildInputs`.